gapic-generator/gapic-generator.gemspec

```diff
@@ -46,6 +46,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "rubocop", "~> 0.61"
 
   spec.add_development_dependency "bundler", "~> 1.17"
+  spec.add_development_dependency "grpc-tools", "~> 1.19"
```

Comment:
don't we need this as a runtime dependency?

Reply:
Not AFAICT. The generator shouldn't mandate how it is used. Users should be able to use the generator by either installing `protoc` using `apt-get`, or compiling it, or installing `grpc-tools` and calling `grpc_tools_ruby_protoc`.

Reply:
I suppose that's true. Our first goal is to be a protoc plugin and you're totally right about not mandating in that respect so let's keep it like this.